### ISSUE TITLE
Create and publish a good email and assign someone (Sec or Pres) to respond.

### ISSUE THREAD

**Issue by RobertLRead** [presidential-innovation-foundation/presidential-innovation-foundation.github.io#36]

**Comment by ultrasaurus** [comment]
The approach we take in RailsBridge which has worked really well is to have a hello@  and then create a Google Group at that address with a few people on it who can respond.  I would be happy to be one of the people, but not the only one!  

totally open to a different incoming name.  

Also, might be good to have board@ to go to the whole board, in case there's something official or the membership wants to address the whole board (& for our ease of emailing)

cc @benbalter

**Comment by ultrasaurus** [comment]
Looping in @polastre who pointed this out in case he has thoughts on a name for incoming PIF Foundation inquiries

**Comment by benbalter** [comment]
We have `hello@` which now goes to the board. Seems like it's just a matter of updating the site to publish that? If that's the case, will submit a PR momentarily.

**Comment by RobertLRead** [comment]
@benbalter Ben I think this REALLY needs to be done.  I don't actually know how to do it --- that is, I technically, don't know how to make the email work.  But I think this is very urgent, and I suspect you, Ben, know how to do it better than I.  If you don't know how to do it, passing it back

**Comment by RobertLRead** [comment]
@benbalter This was a little freaky --- I was writing the comment above write as you posted this!

**Comment by RobertLRead** [comment]
@benbalter Yes, I think we should just update the site to publish that as a contact email !!! Please do it, thanks!!!

**Comment by benbalter** [comment]
Email is live, pull request to document it within the site over in https://github.com/presidential-innovation-foundation/presidential-innovation-foundation.github.io/pull/39.

**Comment by RobertLRead** [comment]
@benbalter I don't think the email visual stands out---can we change the color or put the angle brackets around it like <read.robert@gmail.com> for example?  Or is that out of fashion?
